### PR TITLE
testmap: Update cockpit-composer test

### PR DESCRIPTION
**Pull request by henrywang**
- Add fedora-33/firefox and remove fedora-32/firefox
- Add rhel-8-4/firefox
- Remove rhel-8-4/firefox from manual

### CODE REVIEW

File: task/testmap.py [hunk 1]
```diff
@@ -124,17 +124,17 @@
     'osbuild/cockpit-composer': {
         'master': [
             'fedora-32',
-            'fedora-32/firefox',
             'fedora-33',
+            'fedora-33/firefox',
```
Comment (marusak) [inline task/testmap.py:128]: This is a new context, so you cannot trigger it in this PR.
It first needs to go into `_manual`, then you can test it and then we can move it to `master`.

Comment (henrywang) [reply task/testmap.py]: Make sense to me. Thanks @marusak!

**Review by marusak — APPROVED** [review]
Thanks!
Let's wait for `rhel-8-4/firefox` result. (or if you have some run by hand, please post it here)
When this is merged, please send another PR where you move `fedora-33/firefox` to `master` ideally with test result, but if you believe it will just work, then feel free to just run it in the PR itself.

**Comment by henrywang** [comment]
@marusak, Thanks. I'll send another PR to move `fedora-33/firefox` to `master`. Let's land this PR first.

**Pull request merged**